### PR TITLE
Created a formatPrice util

### LINKED ISSUE #129: Create formatPrice utility
Convert number into a currency
image

### PR DESCRIPTION
**Pull request by D1nviz**
Converts number value into the correct currency
image
image

### CODE REVIEW

File: code/src/utils/formatPrice.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const formatPrice = (
+  price: number,
+  options: {
+    currency?: "USD" | "EUR" | "GBP" | "BDT";
```
Comment (mhevyk) [inline code/src/utils/formatPrice.ts]: Move to separate type for better readability

Comment (D1nviz) [reply code/src/utils/formatPrice.ts]: Moved

File: code/src/utils/formatPrice.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+type FormatPriceOptions = {
+  currency?: "USD" | "EUR" | "GBP" | "BDT";
```
Comment (mhevyk) [inline code/src/utils/formatPrice.ts]: What about separate type for currency? I like to move union types like variant into separate type

Comment (D1nviz) [reply code/src/utils/formatPrice.ts]: That's what you mean? Refactored

**Review by mhevyk — APPROVED** [review]

**Review by heshvab — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**